README.md

```diff
@@ -1283,6 +1283,7 @@ See **[Ticketing](#ticketing)**
 - [RequestBin](https://github.com/Runscope/requestbin) - Inspect HTTP requests. Debug webhooks. `MIT` `python`
 - [RStudio Server](https://www.rstudio.com/products/rstudio/#Server) - Web browser based IDE for R. ([Source Code](https://github.com/rstudio/rstudio)) `AGPL-3.0` `Java/C++`
 - [sourcegraph](https://sourcegraph.com) - Sourcegraph is a fast, open-source, fully-featured code search and navigation engine written in Go. ([Source Code](https://github.com/sourcegraph/sourcegraph)) `Apache-2.0` `Go`
+- [Theia](https://github.com/eclipse-theia/theia) - Theia is a cloud & desktop IDE with a Vscode Editor for the browser.
```

Comment:
It uses VS Code as a base but it is not really vscode for the browser. Extensions do not work and they build their own "plugins". Code-server is vscode in the browser.